### PR TITLE
Tkt 00199 - Call to action cards

### PR DESCRIPTION
**Pull request by Android63**
Add styling for CTA cards as shown below (except borders will be blue):

screen shot 2018-05-11 at 17 33 33

### CODE REVIEW

File: components/cards/_components.cards.scss [hunk 1]
```diff
@@ -29,3 +29,14 @@
...
+
+.c-card--information > h4 {
+	line-height: 1;
+	margin-bottom: $global-border-mid;
+}
+.c-card--information small {
+	margin-bottom: $global-border-mid;
+}
+.c-card--information p {
+	margin-bottom: 0;
```
Comment (cehwitham) [inline components/cards/_components.cards.scss]: Does this cause issues if people add more than one paragraph of content?

File: components/cards/_components.cards.scss [hunk 2]
```diff
@@ -29,3 +29,14 @@
 .c-card--information {
   border-color: $color-information;
 }
+
+.c-card--information > h4 {
+	line-height: 1;
```
Comment (cehwitham) [inline components/cards/_components.cards.scss]: I don't think we should be setting line-heights outside of the main mixin? What's the need here?

File: components/cards/_components.cards.scss (hunk 1 above)
Comment (Android63) [reply components/cards/_components.cards.scss]: Yes. It eliminates the line break between paragraphs. I've fixed this by targeting the first p element only. Result is as follows:

screen shot 2018-05-15 at 17 45 42

File: components/cards/_components.cards.scss (hunk 2 above)
Comment (Android63) [reply components/cards/_components.cards.scss]: This is to allow headers to wrap without too much space between lines. But maybe this is a general requirement. What do you think @Pro-Paul?

Comment (Android63) [reply components/cards/_components.cards.scss]: Any suggestions @Pro-Paul? I'm wondering if we should set the line height for all headers.

Comment (penpadstudio) [reply components/cards/_components.cards.scss]: @cehwitham @Android63 I have already highlighted the issue of line heights that needs to be addressed and upon updating the code to solve the issue @cehwitham has requested assistance from @csswizardry to improve the way the code is written for line heights: https://github.com/NHSLeadership/nightingale/pull/211

If this is solved then I think this question would then become redundant.

**Comment by Android63** [comment]
I have simplified this quite a lot. The result is as shown below:

screen shot 2018-05-21 at 15 37 40

Could you review again please @cehwitham?

**Review by cehwitham — APPROVED** [review]

**Pull request merged**